### PR TITLE
Remove and reclone repo if URL has changed.

### PR DESCRIPTION
**Pull request by aaron-kaplan**
...plus some fixes to unrelated problems I noticed while working on it. Please read commit-by-commit.

This can wait until after PR #3. I'll rebase and resolve conflicts if necessary.

### CODE REVIEW

**Comment by aaron-kaplan** [comment]
@kgraaf

File: roles/iridl/templates/update_datalib [hunk 1]
```diff
@@ -17,8 +17,24 @@ init_git() {
     local target=$2
     local version=$3
 
-    mkdir -p $target
-    if [ ! -d $target/.git ]; then
+    # Check if $target is a valid clone of the right repo. If not, remove it.
+    if [[ -e $target ]]; then
+        if remotes=$(cd $target && git remote -v); then
+            # extract the remote url and compare it to $repo
+            echo $repo
```
Comment (kgraaf) [inline roles/iridl/templates/update_datalib:24]: remove printf debugging

**Review by kgraaf — CHANGES_REQUESTED** [review]

**Comment by kgraaf** [comment]
make sure this doesn't semantically conflict with master, since the update_script has been reordered since this PR was created

**Comment by aaron-kaplan** [comment]
@kgraaf I've removed the debugging prints, rebased, and retested.

**Review by kgraaf — APPROVED** [review]

**Pull request merged**